pandas/plotting/_core.py

```diff
@@ -2721,18 +2721,81 @@ def line(self, x=None, y=None, **kwds):
 
     def bar(self, x=None, y=None, **kwds):
         """
-        Vertical bar plot
+        Vertical bar plot.
+
+        A bar plot is a plot that presents categorical data with
+        rectangular bars with lengths proportional to the values that they
+        represent. A bar plot shows comparisons among discrete categories. One
+        axis of the plot shows the specific categories being compared, and the
+        other axis represents a measured value.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, default None
+            Allows plotting of one column versus another.
+        y : label or position, default None
+            Allows plotting of one column versus another.
+        **kwds : optional
             Keyword arguments to pass on to :py:meth:`pandas.DataFrame.plot`.
```

Comment:
Can you rephrase this as ``Additional keyword arguments are documented in meth:`pandas.DataFrame.plot` `` ?